cmd/ebrelayer/main.go

```diff
@@ -204,8 +205,13 @@ func RunInitRelayerCmd(cmd *cobra.Command, args []string) error {
 		sugaredLogger,
 	)
 
+	key, err := txs.LoadPrivateKey()
```

Comment:
Adding the private key here since CosmosSub has always had a PrivateKey field, and NewCosmosSub should take that key